system_setup/cmd/tui.py

```diff
@@ -133,6 +121,10 @@ def main():
     logger.info("Starting System Setup revision {}".format(version))
     logger.info("Arguments passed: {}".format(sys.argv))
 
+    opts.ssh = False
+    opts.screens = None
+    opts.scripts = None
```

Comment:
Default for ssh is False.
Default for screens and scripts are []. Shouldn’t we use those default values rather than relying on the "falsy" behavior of python?

Reply:
That's true...